conversion/tempurator-convertor

```diff
@@ -0,0 +1,115 @@
+# temperataure converter
+
+# units of temperature
+print("""
+Units of temperature are : 
+
+1. celsius
+2. fahrenheit
+3. kelvin
+4. rankine
+5. delisle
+6. newton""")
+
+# use converter
+
+a = input("To use converter press 'c' OR to close the program press any key   key = ")
```

Comment:
you can add the helper function in this program

Reply:
I have made some helpful changes in the code. But please tell me what is helper function.